app/assets/stylesheets/templates/_home.scss

```diff
@@ -216,45 +216,110 @@
     }
   }
 
-  .slab--testimonials {
-    background-color: white;
-
+  .slab--rating {
+    padding-bottom: 3.5rem;
+    .title {
+      font-size: 2.7rem;
+    }
+    .text--small {
+      font-size: 1.4rem;
+    }
     #star-rating {
-      margin-bottom: 4rem;
+      img {
+        height: 4rem;
+        width: 100%;
+      }
+      @media screen and (max-width: $tablet-up) {
+        padding: 0;
+      }
     }
-
-    #testimonial-left {
-      padding-right: 6rem;
+    #trusted-support {
+      padding-left: 3rem;
+      @media screen and (max-width: $tablet-up) {
+        text-align: center;
+        padding: 0 1.5rem 0 1.5rem;
+      }
     }
+  }
 
-    #testimonial-right {
-      padding-left: 6rem;
+  .slab-habla-espanol {
+    padding-top: 0;
```

Comment:
I noticed that the h2 in the mock has less padding between the p than the implementation does. I think that's probably fine instead of overriding the h2 spacing, though.